src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -1021,12 +1021,22 @@ void runtime::gpu::GPU_Emitter::emit_Reduce(EMIT_ARGS)
             }
             else
             {
-                // in current implementation:
-                // 1. reduction function should only have one op
-                // 2. the op should be in the op_map
-                // otherwise, throw an error message
-                cudnnReduceTensorOp_t reduce_tensor_op;
+                auto axes_set = reduce_op->get_reduction_axes();
+                ngraph::AxisVector axes_vec;
+                for (auto a : axes_set)
+                {
+                    axes_vec.push_back(a);
+                }
+                vector<string> dtypes;
```

Comment:
nit: std::vector, to be consistent with the rest of your standard lib use.

Reply:
fixed.